sdk/src/events/eventSubscriber.ts

```diff
@@ -57,8 +58,32 @@ export class EventSubscriber {
 				this.connection,
 				this.address,
 				this.options.commitment,
-				this.options.resubTimeoutMs
+				this.options.logProviderConfig.resubTimeoutMs
 			);
+			if (this.options.logProviderConfig.resubTimeoutMs) {
```

Comment:
nit: should this be in the subscribe fn? or maybe even in the websocket log provider?